src/main/java/org/springframework/hateoas/MediaTypes.java

```diff
@@ -34,4 +35,14 @@
 	 * Public constant media type for {@code application/hal+json}.
 	 */
 	public static final MediaType HAL_JSON = MediaType.valueOf(HAL_JSON_VALUE);
+	
+	/**
+	 * A String equivalent of {@link MediaTypes#HAL_JSON_UTF8}.
+	 */
+	public static final String HAL_JSON_UTF8_VALUE = HAL_JSON_VALUE + ";charset=UTF-8";
```

Comment:
Indeed, I missed that "APPLICATION_JSON_UTF8_VALUE" constant in MediaType... It's weird that they chose to write it like that, since Guava / Spring / JDK seem to prefer the underscore everywhere else (I even cloned a few Spring project and grep'ed to check :D ). 

I guess it makes sense in a way, since you clearly separate the "HAL" / "JSON" and "UTF 8" info. We should probably go for consistency with MediaTypes.

Anyway I created a temporary class "AdditionalMediaTypes" in my project, until we can get this in Spring HATEOAS. Great suggestion :)